### PR TITLE
DA-1279: Enable pylint

### PR DESCRIPTION
**Pull request by j-kanuch**
Enabling PyLint resulted in a number of warnings and errors that needed to be resolved before pushing. These are summarized as follows:
  - Fixed whitespace warnings.
  - Disabled PyLint on unused imports, unused variables, and unused arguments.
  - Disabled or fixed undefined variable errors.
  - Removed the 'rdr_service/test' directory from PyLint.

### CODE REVIEW

File: tests/api_tests/test_genomic_set_file_handler.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 )
 from rdr_service.model.participant import Participant
 from tests import test_data
-from rdr_service.storage import LocalFilesystemStorageProvider
+from rdr_service.storage import LocalFilesystemStorageProvider  # pylint: disable=unused-import
```
Comment (Seeker1911) [inline tests/api_tests/test_genomic_set_file_handler.py]: @wangy70 any reason to import this specifically here? BaseTestCase uses it internally. I think we can remove this reference.

File: tests/client_tests/base.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import copy
 import json
 import os
-import unittest
+import unittest  # pylint: disable=unused-import
```
Comment (Seeker1911) [inline tests/client_tests/base.py]: remove line

File: tests/dao_tests/test_ehr_dao.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import datetime
-import unittest
+import unittest  # pylint: disable=unused-import
```
Comment (Seeker1911) [inline tests/dao_tests/test_ehr_dao.py]: remove line

File: tests/dao_tests/test_ehr_dao_test.py [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import datetime
-import unittest
+import unittest  # pylint: disable=unused-import
```
Comment (Seeker1911) [inline tests/dao_tests/test_ehr_dao_test.py]: remove line

File: tests/test_database.py [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import datetime
 import isodate
-import unittest
+import unittest  # pylint: disable=unused-import
```
Comment (Seeker1911) [inline tests/test_database.py]: remove line

File: rdr_service/dao/questionnaire_dao.py [hunk 6]
```diff
@@ -43,6 +43,8 @@ def get_latest_questionnaire_with_concept(self, codeId):
             )
 
     def _make_history(self, questionnaire, concepts, questions):
+        # pylint: disable=redefined-outer-name
+        # TODO: refactor variable 'questionnaire' as it is defined in outer scope (line 9)
```
[the remark this answers is not in the document]
Comment (j-kanuch) [reply rdr_service/dao/questionnaire_dao.py]: Line removed in next commit.

File: rdr_service/dao/questionnaire_dao.py [hunk 7]
```diff
@@ -61,6 +63,8 @@ def _make_history(self, questionnaire, concepts, questions):
         return history
 
     def insert_with_session(self, session, questionnaire):
+        # pylint: disable=redefined-outer-name
+        # TODO: refactor variable 'questionnaire' as it is defined in outer scope (line 9)
```
[the remark this answers is not in the document]
Comment (j-kanuch) [reply rdr_service/dao/questionnaire_dao.py]: Line removed in next commit.

File: rdr_service/dao/questionnaire_dao.py [hunk 8]
```diff
@@ -100,6 +104,8 @@ def _do_update(self, session, obj, existing_obj):
         super(QuestionnaireDao, self)._do_update(session, obj, existing_obj)
 
     def update_with_session(self, session, questionnaire):
+        # pylint: disable=redefined-outer-name
+        # TODO: refactor variable 'questionnaire' as it is defined in outer scope (line 9)
```
[the remark this answers is not in the document]
Comment (j-kanuch) [reply rdr_service/dao/questionnaire_dao.py]: Line removed in next commit.

File: rdr_service/genomic/validation.py [hunk 9]
```diff
@@ -1,7 +1,7 @@
 import collections
 import datetime
 import functools
-import itertools
+import itertools  # pylint: disable=unused-import
```
[the remark this answers is not in the document]
Comment (j-kanuch) [reply rdr_service/genomic/validation.py]: Line removed in next commit.

File: rdr_service/main.py [hunk 10]
```diff
@@ -3,7 +3,7 @@
 This defines the APIs and the handlers for the APIs. All responses are JSON.
 """
 import logging
-import os
+import os  # pylint: disable=unused-import
```
[the remark this answers is not in the document]
Comment (j-kanuch) [reply rdr_service/main.py]: Line removed in next commit.

File: rdr_service/model/metrics.py [hunk 11]
```diff
@@ -1,4 +1,4 @@
-from sqlalchemy import BLOB, Boolean, Column, Date, ForeignKey, Integer, String
```
[the remark this answers is not in the document]
Comment (j-kanuch) [reply rdr_service/model/metrics.py]: BLOB import added back into file.

File: rdr_service/model/participant.py [hunk 12]
```diff
@@ -1,5 +1,4 @@
 from sqlalchemy import (
-    BLOB,
```
[the remark this answers is not in the document]
Comment (j-kanuch) [reply rdr_service/model/participant.py]: BLOB import added back into file.

File: rdr_service/model/questionnaire.py [hunk 13]
```diff
@@ -1,4 +1,12 @@
-from sqlalchemy import BLOB, Boolean, Column, ForeignKey, ForeignKeyConstraint, Integer, String, UniqueConstraint
```
[the remark this answers is not in the document]
Comment (j-kanuch) [reply rdr_service/model/questionnaire.py]: BLOB import added back into file.

File: rdr_service/model/questionnaire_response.py [hunk 14]
```diff
@@ -1,4 +1,14 @@
-from sqlalchemy import BLOB, Boolean, Column, Date, Float, ForeignKey, ForeignKeyConstraint, Integer, String, Text
```
[the remark this answers is not in the document]
Comment (j-kanuch) [reply rdr_service/model/questionnaire_response.py]: BLOB import added back into file.

File: rdr_service/offline/metrics_pipeline.py [hunk 15]
```diff
@@ -123,6 +123,7 @@
 # from mapreduce import mapreduce_pipeline
 # from mapreduce import context
 
+# pylint: disable=undefined-variable
```
[the remark this answers is not in the document]
Comment (j-kanuch) [reply rdr_service/offline/metrics_pipeline.py]: The following undefined variables were causing pylint error messages:
- base_handler
- GCSInputReader
- mapreduce_pipeline
- context

File: rdr_service/tools/build_cron_yaml.py [hunk 16]
```diff
@@ -9,109 +9,109 @@
 import argparse
 import collections
 import os
-import sys
+import sys  # pylint: disable=unused-import
```
[the remark this answers is not in the document]
Comment (j-kanuch) [reply rdr_service/tools/build_cron_yaml.py]: Line removed in next commit.

File: rdr_service/tools/csv_importer.py [hunk 17]
```diff
@@ -3,7 +3,7 @@
 import csv
 import logging
 
-from rdr_service.unicode_csv import UnicodeDictReader
+from rdr_service.unicode_csv import UnicodeDictReader  # pylint: disable=unused-import
```
[the remark this answers is not in the document]
Comment (j-kanuch) [reply rdr_service/tools/csv_importer.py]: Line removed in next commit.

File: rdr_service/tools/process_ptsc_pairings.py [hunk 18]
```diff
@@ -1,7 +1,7 @@
 import csv
 
 from rdr_service.main_util import configure_logging, get_parser
-from rdr_service.unicode_csv import UnicodeDictReader
+from rdr_service.unicode_csv import UnicodeDictReader  # pylint: disable=unused-import
```
[the remark this answers is not in the document]
Comment (j-kanuch) [reply rdr_service/tools/process_ptsc_pairings.py]: Line removed in next commit.

File: rdr_service/version_api.py [hunk 19]
```diff
@@ -3,7 +3,7 @@
 No auth is required for this endpoint because it serves nothing sensitive.
 """
 
-import os
+import os  # pylint: disable=unused-import
```
[the remark this answers is not in the document]
Comment (j-kanuch) [reply rdr_service/version_api.py]: Line removed in next commit.

File: tests/client_tests/base.py (hunk 2 above)
Comment (j-kanuch) [reply tests/client_tests/base.py]: Line removed in next commit.

File: tests/dao_tests/test_ehr_dao.py (hunk 3 above)
Comment (j-kanuch) [reply tests/dao_tests/test_ehr_dao.py]: Line removed in next commit.

File: tests/dao_tests/test_ehr_dao_test.py (hunk 4 above)
Comment (j-kanuch) [reply tests/dao_tests/test_ehr_dao_test.py]: Line removed in next commit.

File: tests/test_database.py (hunk 5 above)
Comment (j-kanuch) [reply tests/test_database.py]: Line removed in next commit.

File: tests/api_tests/test_genomic_set_file_handler.py (hunk 1 above)
Comment (j-kanuch) [reply tests/api_tests/test_genomic_set_file_handler.py]: Line removed in next commit.

**Comment by j-kanuch** [comment]
Updated tests/helpers/mysql_helper.py for merge conflicts.

**Pull request merged**